### ISSUE TITLE
[a11y] Notification downloads should be more pronounced

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1635]
![image](https://user-images.githubusercontent.com/37256050/168871257-212e7d9d-e4c1-4ada-b6f3-51743e49dcde.png)

The Download option should be a button or at least bold text. Having it on a separate line would be a good idea.

**Comment by maxxxxxdlp** [comment]
Fixed in 5a7055d9 by moving the button to a separate line
